CONTRIBUTING.md

```diff
@@ -96,6 +96,30 @@ package (PyQt5). This can be achieved by running:
   $ pip install pyqt5
 ```
 
+### Using the correct version of spyder-kernels
+
+Following the separation of ``spyder-kernels`` into its own modular package
```

Comment:
> You don't need double backticks in markdown, just one (double backticks is a rst stuff :-). So you can replace ```` for `` in all places in this PR.

Yup, I'm well aware :) I've very deliberately standardized on double backtick for both ``rst`` and ``md`` as stated in the (to be published shortly) official style guide and already implemented in all the docs and other ``rst`` and ``md`` files across this repo as recommended in various best practices, as it promotes interoperability between rst and Markdown files and text snippits and makes conversion easier, avoids accidentally using the wrong syntax in the wrong parser (as was evident numerous places in our previous docs before my big overhaul, where single backtick was used where double was clearly intended), helps build a common muscle memory increasing typing speed (offsetting any small decrease from the double- vs single-tap, and promotes a common, consistent convention across the entire repo, with no real downsides.

> i don't like this wording

Changed to 

> Following the separation in v3.3 of Spyder's console code into its own package,
``spyder-kernels``, ...

Reply:
In fact, the latter was *particularly* convenient here since I could use the ``rst`` snippit from the docs almost verbatim (except for that pesky link that I missed :).

Reply:
> Yup, I'm well aware :) I've very deliberately standardized on double backtick for both rst and md

I don't like that. Please revert it. There are two reasons for it

1. It really makes no sense to depart from regular markdown just so that rst and markdown follow the same style. Both are different markup languages. This will be odd also for new comers too.
2. Writing ``` is really cumbersome in Latin American keyboards, so the less we have to do it, the better.

Reply:
> Changed to

    Following the separation in v3.3 of Spyder's console code into its own package,
    spyder-kernels, ...

Great, thanks.

Reply:
> I don't like that. Please revert it. 

Okay, done. Part of the reason I was more insistent before is I got confused and thought this was the PR on the docs repo instead of the main one, where the case for it is much stronger.

> It really makes no sense to depart from regular markdown just so that rst and markdown follow the same style. 

I explained the rationale for this above; this is particularly helpful here since the same or similar text is featured in both places, so interoperability is particularly beneficial for reducing the maintenance burden.

> Writing ` is really cumbersome in Latin American keyboards, so the less we have to do it, the better.

Ah, I see—I was unaware. My reasoning was based off that on a standard US keyboard, there's a dedicated `` ` `` key and so while getting my little finger to it takes some time, double-tapping it is almost as fast as single-tapping. As this is evidently very much not the case for some other layouts, this doubling up indeed makes much less sense.